common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -155,6 +163,11 @@ def assert_text_in_notes(self, notes):
         expected = ["TEST TEXT {}".format(i) for i in xrange(len(notes))]
         self.assertEqual(expected, actual)
 
+    def assert_tags_in_notes(self, notes, expected_tags):
+        actual = [note.tags for note in notes]
```

Comment:
Same reasoning as above.

In general, I find having to add docstrings on every method annoying. There are certainly times they are useful (especially if you need to describe a bunch of input args or expected return value format), but I prefer to use descriptive method names and variable names, as those are more likely to remain correct. Documentation always rots.